actors/builtin/reward/reward_logic.go

```diff
@@ -5,8 +5,30 @@ import (
 	big "github.com/filecoin-project/specs-actors/actors/abi/big"
 )
 
-var BaselinePowerAt = func(epoch abi.ChainEpoch) abi.StoragePower {
-	return big.NewInt(1 << 40) // PARAM_FINISH
+const BaselineExponentString = "340282663082994238536867392845056089438"
+
+// Baseline function = BaselineInitialValue * (BaselineExponent) ^(t), t in epochs
+var BaselineExponent big.Int     // Q.128
+var BaselineInitialValue big.Int // Q.0
+
+func init() {
+	BaselineExponent = big.MustFromString(BaselineExponentString)
+	BaselineInitialValue = big.Lsh(big.NewInt(1), 60) // 1 EiB
```

Comment:
@zixuanzh can you confirm this is the desired value?

Reply:
yes - this is our preliminary recommendation